src/zunit.zsh

```diff
@@ -31,7 +31,7 @@ function _zunit_usage() {
 # Output the version number
 ###
 function _zunit_version() {
-  echo '0.8.1'
+  echo '0.9.0'
```

Comment:
the version here is incosistent with the PR. Could you update either this or the PR title?

Reply:
Good catch!